debian/control

```diff
@@ -2,7 +2,7 @@ Source: kodi-pvr-nextpvr
 Priority: extra
 Maintainer: Nobody <nobody@kodi.tv>
 Build-Depends: debhelper (>= 9.0.0), cmake, libtinyxml-dev,
-               libkodiplatform-dev (>= 16.0.0), kodi-addon-dev
+               kodi-addon-dev
```

Comment:
Shit, there then need to add `libp8-platform-dev`, Sorry that I'm not seen before :sweat_smile:.

Can you add here on your other request the `libp8-platform-dev`?

Reply:
I think to look how debian build can be included inside travis or AppVeyor.

Trying now request about.

Reply:
@AlwinEsch I built this patch without p8 using debian-development VM (git-pbuilder etc)